project-koku/koku
# [COST-5618] Remove schema size prometheus metric reporting

## Jira Ticket

[COST-5618](https://issues.redhat.com/browse/COST-5618)

## Description

* Prometheus metric has too high a cardinality, so we need to remove this

## Testing

1. Checkout Branch
2. Restart Koku
3. Load test data to populate schema size
4. Hit koku-clowder-worker-celery metrics endpoint and note the postgresql_schema_size_bytes metric is no longer present

## Release Notes
- [ ] Remove schema size prometheus metric reporting

```markdown
* [COST-5618](https://issues.redhat.com/browse/COST-5618) Remove schema size prometheus metric reporting
```
